src/js/common/schemaValidatedComponent.js

```diff
@@ -0,0 +1,310 @@
+/* globals require */
+(function (fluid) {
+    "use strict";
+
+    if (typeof require !== "undefined") {
```

Comment:
This is good enough for now but I imagine might anger some people down the line who might eventually want to distribute infusion/this library/others using webpack or some similar scheme. I don't currently know of any such people. Might it not be safer just to sniff if fluid itself is defined as we do elsewhere?

Reply:
I updated to check for the presence of fluid instead.